notion/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0a709534-658c-4d8f-99a3-566dd7cd809b",
+  "app_id": "notion",
+  "display_on_public_website": true,
```

Comment:
I know there has been some back and forth on this field, but I want to clarify what it means when you are planning on shipping to a select set of customers first:

- If this field is set to true, then once this PR is merged the Notion integration will appear in our official Datadog docs (but will only appear on the Integrations page within the DD app to the approved customers that are part of the EAP)
- If this field is set to false, then Notion will *not* show up in the official Datadog docs once this PR is merged, but selected EAP customers will see it on the Integrations page within the DD app. Once you're ready to go GA, you'd open a PR to change this value to true so we can show it in our docs.

Which way would you like to proceed?

Reply:
set to false, we'll open another pr to change it